src/tools/tests/schema2kotlin-test.ts

```diff
@@ -256,4 +256,53 @@ describe('schema2kotlin', () => {
     const components = schema2kotlin.generateParticleClassComponents(particle, generators);
     assert.deepEqual(extractor(components), expectedValue);
   }
+
+  describe('Test Harness', () => {
+    it('exposes a handle as a read write handle regardless of particle spec direction', async () => await assertTestHarness(
+      `particle P
+        h1: reads Person {name: Text}
```

Comment:
What about adding a second handle that is a write connection?

Reply:
Done